TodoListService-ManualJwt/Global.asax.cs

```diff
@@ -78,22 +78,12 @@ internal class TokenValidationHandler : DelegatingHandler
         //
         protected async override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
-            string authHeader = null;
+            // Get the jwt bearer token from the authorization header
             string jwtToken = null;
-            string issuer;
-            string stsDiscoveryEndpoint = string.Format("{0}/.well-known/openid-configuration", authority);
-
-            List<SecurityToken> signingTokens;
-
-            // The header is of the form "bearer <accesstoken>", so extract to the right of the whitespace to find the access token.
-            authHeader = HttpContext.Current.Request.Headers["Authorization"];
+            AuthenticationHeaderValue authHeader = request.Headers.Authorization;
             if (authHeader != null)
             {
-                int startIndex = authHeader.LastIndexOf(' ');
```

Comment:
String manipulation isn't required, as there is strongly typed access available to known headers